aim/db/model_base.py

```diff
@@ -65,6 +65,11 @@ class HasTenantName(object):
     tenant_name = name_column(nullable=False)
 
 
+class IsManaged(object):
+    """Add to subclasses that are manager objects"""
```

Comment:
nit: manager -> managed

Also might just want to call this call "IsMonitored" instead because monitored objects are actually not being managed by AIM/AID